XCUITests/SearchTest.swift

```diff
@@ -0,0 +1,167 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import XCTest
+
+private let LabelPrompt = "Turn on search suggestions?"
+private let SuggestedSite = "foobar2000.org"
+
+class SearchTests: BaseTestCase {
+    var navigator: Navigator!
+    var app: XCUIApplication!
+    
+    override func setUp() {
+        super.setUp()
+        app = XCUIApplication()
+        navigator = createScreenGraph(app).navigator(self)
+        dismissFirstRunUI()
+    }
+    
+    override func tearDown() {
+        super.tearDown()
+    }
+    
+    private func suggestionsOnOff() {
+        navigator.goto(SearchSettings)
+        app.tables.switches["Show Search Suggestions"].tap()
+        app.navigationBars["Search"].buttons["Settings"].tap()
+        app.navigationBars["Settings"].buttons["AppSettingsTableViewController.navigationItem.leftBarButtonItem"].tap()
+    }
+    
+    private func typeOnSearchBar(text: String) {
+        app.textFields["url"].tap()
+        app.textFields["address"].typeText(text)
+    }
+    
+    func testPromptPresence() {
+        // Suggestion is off by default, so the prompt should appear
+        typeOnSearchBar(text: "foobar")
```

Comment:
That option does not work for this test, the url has not to be open, it is needed just to write in the search bar to see if the prompt to enable suggestions is shown. If the url is open directly, this cannot be checked